app/models/nav.rb

```diff
@@ -12,9 +12,7 @@ def current_item_root
 
   # Returns the current nav item
   def current_item
-    item = route_map[path.sub("/docs/", "")]
-    raise ActionController::RoutingError.new("Missing navigation for #{path}") unless item
```

Comment:
I found this error useful for handling orphaned pages. I think we should instead add a condition for the home page separately.

e.g

```
return nil if path =="/"
```

Reply:
Ah I see. I just pushed up a fix [here](https://github.com/buildkite/docs/pull/2054/files#diff-1788a142b31ad28644197424968d69a9d0b8ffca46a37605dfdaaf74345fd409R15)

```
return nil if path == "/docs"
```